lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -12,6 +12,7 @@
 #include "mlir/IR/BuiltinDialect.h"
 #include "mlir/Transforms/DialectConversion.h"
 #include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+#include "torch-mlir/Conversion/Utils/Utils.h"
```

Comment:
nit: I don't think this is being used?

Reply:
This isn't being used by my patch, but I think that clang-format moved this (the build fails without this import)